test/patch/registerElement.spec.js

```diff
@@ -129,4 +129,27 @@ describe('document.registerElement', ifEnvSupports(registerElement, function ()
     var elt = document.createElement('x-props-desc');
   });
 
+
+  it('should check bind callback if not own property', function (done) {
+    testZone.run(function() {
+      var originalProto = {
+        createdCallback: function() {}
+      }
+      var secondaryProto = Object.create(originalProto);
+      expect(secondaryProto.createdCallback).toBe(originalProto.createdCallback);
+      expect(function() {
+        document.registerElement('x-inherited-callback', { prototype: secondaryProto });
+        expect(secondaryProto.createdCallback).not.toBe(originalProto.createdCallback);
+        done();
```

Comment:
`done` should not be required as of now, the test is sync. 

However I think it would be great to actually create an element an assert the cb zone.